extension/src/languageServer/languageServer.ts

```diff
@@ -53,30 +54,24 @@ export async function startLanguageServer(
                 serverOptions = awaitServerConnection.bind(null, port);
             } else {
                 // keep consistent with gRPC server
-                const javaHome = getConfigGradleJavaHome() || process.env.JAVA_HOME;
+                const javaHome = getJavaHome();
+                let javaCommand;
                 if (!javaHome) {
```

Comment:
Just 2 cents, flip the if-else branch would be more human-readable for me, because you don't assign a new `javaHome` in `if !javaHome` block..

Below makes more sense to me:

a) 
```
if (!javaHome) {
    javaHome = .....
    // do something requiring javaHome
} else {
    // do something requiring javaHome
}
```


b) 
```
if (javaHome) {
    // do something requiring javaHome
} else {
    // do something unrelated with javaHome
}
```